lib/SynchronousHandover.ts

```diff
@@ -1,10 +1,11 @@
 import { IModify, IPersistence, IRead } from '@rocket.chat/apps-engine/definition/accessors';
 import { IApp } from '@rocket.chat/apps-engine/definition/IApp';
 import { ILivechatRoom } from '@rocket.chat/apps-engine/definition/livechat';
-import { AppSetting } from '../config/Settings';
+import { AppSetting, DefaultMessage } from '../config/Settings';
```

Comment:
Where do you use 'DefaultMessage'?

Reply:
I missed to remove this after fixing Conflicts, Thanks for pointing Eric.
I'll remove this